tests/plugins/font/customfontdefinition.js

```diff
@@ -7,40 +7,69 @@
 ( function() {
 	'use strict';
 
-	bender.editor = {
-		config: {
-			extraAllowedContent: 'span font(*){*}[*];strong em;',
-			font_style: {
-				element: 'font',
-				attributes: { 'face': '#(family)' },
-				overrides: [ {
-					element: 'font', attributes: { 'face': null }
-				} ]
-			},
-			fontSize_sizes: '1/1;2/2;3/3;4/4;5/5;6/6;7/7;',
-			fontSize_style: {
-				element: 'font',
-				attributes: { 'size': '#(size)' },
-				overrides: [ {
-					element: 'font', attributes: { 'size': null }
-				} ]
+	var classAttributes = {},
+		dataAttributes = {};
+
+	classAttributes[ 'class' ] = '#(size)';
+	dataAttributes[ 'data-font' ] = '#(size)';
+
+	bender.editors = {
+		font: {
+			config: {
+				extraAllowedContent: 'span font(*){*}[*];strong em;',
+				font_style: {
+					element: 'font',
+					attributes: { 'face': '#(family)' },
+					overrides: [ {
+						element: 'font', attributes: { 'face': null }
+					} ]
+				},
+				fontSize_sizes: '1/1;2/2;3/3;4/4;5/5;6/6;7/7;',
+				fontSize_style: {
+					element: 'font',
+					attributes: { 'size': '#(size)' },
+					overrides: [ {
+						element: 'font', attributes: { 'size': null }
+					} ]
+				}
+			}
+		},
+
+		classes: {
+			config: {
+				fontSize_sizes: 'small;medium',
+				fontSize_style: {
+					element: 'span',
+					attributes: classAttributes
+				}
+			}
+		},
+
+		attributes: {
+			config: {
+				extraAllowedContent: 'span[data-font]',
+				fontSize_sizes: 'small;medium',
+				fontSize_style: {
+					element: 'span',
+					attributes: dataAttributes
+				}
 			}
 		}
 	};
 
 	bender.test( {
 		'test setdata with font tag with face': function() {
```

Comment:
This test stopped working on IE9 (and only there) with an error
```
Unexpected error: Unable to get value of the property 'html': object is null or undefined
Expected: undefined (undefined)
Actual:   undefined (undefined)
```

So it doesn't seem connected with a fix, but rather with tests refactoring - but nothing important really changed 🤔  Could you please verify if it fails for you too?